src/components/EntryCard.js

```diff
@@ -14,9 +37,52 @@ export const StyledCard = styled.div`
     border: solid 1px;
```

Comment:
I suggest to make every card an 'article', that would be more semantic than a div

Reply:
you´re right. fixed now